Simplified/Simplified.js

```diff
@@ -188,6 +188,31 @@ function Simplified() {
       }";
     innerDocument.head.appendChild(styleElement);
   }
+  
+  /**
+   * setCfiBeforeChange
+   * Before there's a font size or font family change in the UI, keep track of the CFI of
```

Comment:
My only actual request is updating these comments slightly to make it very clear to a future-maintainer their context:
```
FIXME: These two functions are only here provide slightly better support for returning to a particular spot in the book, based on the inconsistency and unreliabiliy of Readium's CFI returning us to the same spot after certain actions like text changing size, etc.
```